src/components/Search/SearchPageHeader/SearchPageHeader.tsx

```diff
@@ -29,7 +29,7 @@ function SearchPageHeader({queryJSON, searchRouterListVisible, hideSearchRouterL
 
     const selectedTransactionsKeys = Object.keys(selectedTransactions ?? {});
 
-    if (shouldUseNarrowLayout && selectionMode?.isEnabled && headerButtonsOptions.length > 0) {
+    if (shouldUseNarrowLayout && !!selectionMode && headerButtonsOptions.length > 0) {
```

Comment:
same case - selection mode is used via `useOnyx` and not via `useSelectionMode`. Why?